requirements.txt

```diff
@@ -23,8 +23,11 @@ greenlet==0.4.16 # pinned to fix build problem (parent is gevent)
 webargs==5.5.3
 ujson==1.33
 requests==2.22.0
-elasticsearch==5.5.3
-elasticsearch-dsl==5.4.0
+# elasticsearch==5.5.3
+# elasticsearch-dsl==5.4.0
```

Comment:
Can you remove the old dependencies please?

Reply:
Obsolete code, forgot to cleanup. Thanks for flagging.